ocaml/libs/tracing/tracing.ml

```diff
@@ -169,7 +169,7 @@ module SpanContext = struct
 end
 
 module SpanLink = struct
-  type t = {context: SpanContext.t; attributes: (string * string) list}
```

Comment:
Can you remove `-warn-error -69` from the file `xen-api.git/dune`? This transforms these warnings into errors, which makes it very difficult to reintroduce these